packages/nextjs/components/scaffold-eth/Balance.tsx

```diff
@@ -1,18 +1,25 @@
+import { useState } from "react";
 import { Address } from "viem";
 import { useAccountBalance } from "~~/hooks/scaffold-eth";
 import { useTargetNetwork } from "~~/hooks/scaffold-eth/useTargetNetwork";
 
 type BalanceProps = {
   address?: Address;
   className?: string;
+  usdMode?: boolean;
 };
 
 /**
  * Display (ETH & USD) balance of an ETH address.
  */
-export const Balance = ({ address, className = "" }: BalanceProps) => {
+export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
   const { targetNetwork } = useTargetNetwork();
-  const { balance, price, isError, isLoading, onToggleBalance, isEthBalance } = useAccountBalance(address);
+  const { balance, price, isError, isLoading, isEthBalance } = useAccountBalance(address);
```

Comment:
we could remove `isEthBalance` from here and also below since we are no more using `onToggleBalance` it was the only function which could change it, so lets just remove it for now